docs/content/contribute/index.md

```diff
@@ -1,48 +1,109 @@
-# Contributing to the Docs
+# 🚀 Contributing to the Docs
 
-We welcome contributions to our documentation! If you would like to contribute, please follow the steps below.
+Welcome to our documentation contribution guide! We're excited to have you here. This guide will help you get started with contributing to our documentation. Let's make our docs better together! 💪
 
-## Setting up the Docs
+<div align="center">
 
-1. Clone the repository:
+![Documentation Contributors](https://img.shields.io/github/contributors/Significant-Gravitas/AutoGPT?style=for-the-badge)
+![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=for-the-badge)
+![Documentation Build Status](https://img.shields.io/badge/docs-passing-success?style=for-the-badge)
 
-    ```shell
-    git clone github.com/Significant-Gravitas/AutoGPT.git
-    ```
+</div>
 
-1. Install the dependencies:
+## 🛠️ Setting up the Docs
 
-    ```shell
-    python -m pip install -r docs/requirements.txt
-    ```
+### Prerequisites
 
-    or
+- Git installed on your machine
+- Python 3.8 or higher
+- A text editor of your choice
 
-    ```shell
-    python3 -m pip install -r docs/requirements.txt
-    ```
+### Step-by-Step Setup Guide
 
-1. Start iterating using mkdocs' live server:
+1. **Clone the Repository** 📥
+   ```shell
+   git clone github.com/Significant-Gravitas/AutoGPT.git
+   ```
 
-    ```shell
-    mkdocs serve
-    ```
+2. **Install Dependencies** 📚
+   ```shell
+   python -m pip install -r docs/requirements.txt
+   ```
+   or
+   ```shell
+   python3 -m pip install -r docs/requirements.txt
+   ```
 
-1. Open your browser and navigate to `http://127.0.0.1:8000`.
+3. **Start the Development Server** 🔥
+   ```shell
+   mkdocs serve
+   ```
 
-1. The server will automatically reload the docs when you save your changes.
+4. **View the Docs** 🌐
+   - Open your browser
+   - Navigate to `http://127.0.0.1:8000`
+   - Changes will auto-reload! 🔄
 
-## Adding a new page
+## 📝 Adding New Content
 
-1. Create a new markdown file in the `docs/content` directory.
-1. Add the new page to the `nav` section in the `mkdocs.yml` file.
-1. Add the content to the new markdown file.
-1. Run `mkdocs serve` to see your changes.
+### Creating a New Page
 
-## Checking links
+1. **Create the File** 📄
+   - Navigate to `docs/content`
+   - Create a new markdown file
```

Comment:
I think you need one more space here for this to render correctly 
![image](https://github.com/user-attachments/assets/78fa7725-fefd-4739-815f-e60d6f1e6520)